src/server/config/auth.js

```diff
@@ -1,11 +1,29 @@
-export function validateAccessTokenFacebook(accessToken) {
-  ///TODO : Validiate accesstoken from facebook API server.
+import request from 'request';
 
-  return false;
+export function crawlByAccessTokenFacebook(accessToken, callback) {
```

Comment:
It's really odd to put the crawler inside auth.js. There should be a dedicated place for crawlers to live. I'm not familiar with the Node.js infrastructure, but wouldn't controller be a better place since we'll be storing these data into user model?